data/human/human missions.txt

```diff
@@ -5299,3 +5299,120 @@ mission "Wool Smuggling 2"
+				decline
+			label late
+			`	He smirks as he walks back towards you. "I'm glad that you've changed your mind."`
+			label listen
+			`	The man pulls you aside to talk. "You see, one of our clients recently passed away, and in his will, he divided his property between each of his relatives. The only problem is that one of his family members, his niece, lives all the way down in the Rim. We would like you to deliver her portion of the inheritance, weighing <tons>, to her residence on <destination>. We can pay you <payment> upon completion of the job."`
+			choice
+				`	"I'll do it."`
+					goto agree
+				`	"What did she inherit?"`
+				`	"Sorry, but I don't want to do this job."`
+					goto nope
+			`	"My sincerest apologies, but I'm afraid that is confidential information. Our prioritization of privacy is one of our main selling points."`
+			choice
+				`	"I'll take the job."`
+				`	"I don't like the idea of carrying an unknown cargo. Count me out."`
+					goto nope
+			label agree
+			`	The man smiles. "I'm glad to hear that. We'll have the cargo loaded onto your ship by sundown." He walks away into the spaceport, leaving you to prepare for the journey.`
+				accept
+			label nope
+			`	The man frowns. "I see. Well, I appreciate that you listened to our offer, and I hope that you may assist us at a later date. Farewell." He walks away into the spaceport but doesn't appear to seek out any other captains.`
+				decline
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		fail "Deal Change 2"
+		payment 100000
+		conversation
+			`You step off your ship and order the spaceport crew to unload your cargo. As you do, a large, fat-lipped woman walks up to you and says in the most exaggerated manner possible, "Oh, thank you so much, captain! You must have gone through so much trouble to help little old me!" She gasps. "Oh, silly old me! I forgot to tell you who I am! I'm the recipient of the inheritance from <origin>, of course! You have truly made a great contribution to the galaxy!" She chuckles.`
+			choice
+				`	"Thank you?"`
+				`	"Could you tell me what the inheritance is?"`
+			`	Before you can get a single word out, the woman exclaims, "I'll be sure to inform your employers of the wonderful work you've done. Ta ta!" She begins to swagger towards a cream-colored car as her inheritance is loaded into a truck beside it. When you return to your ship's cockpit, you find that <payment> have been deposited into your account.`
+
+
+
+mission "Deal Change 2"
+	landing
+	clearance `You read out the code on the back of the paper you received and get a green light from Greenrock's authorities, along with a set of coordinates.`
+	name "Inheritance to <planet>"
+	description "Instead of bringing the inheritance to Skymoot, deliver it to <destination>. You were promised <payment> for doing so."
+	source
+		attributes "dirt belt" "south" "rim"
+		not planet "Greenrock"
+	destination "Greenrock"
```

Comment:
```suggestion
	name "Inheritance to <planet>"
	description "Instead of bringing the inheritance to Skymoot, deliver it to <destination>. You were promised <payment> for doing so."
	source
		attributes "dirt belt" "south" "rim"
		not planet "Greenrock"
	destination "Greenrock"
	clearance `You read out the code on the back of the paper you received and get a green light from Greenrock's authorities, along with a set of coordinates.`
```